Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_ControllerEvents.cs

```diff
@@ -337,6 +315,17 @@ void Update()
                 OnTriggerReleased(SetButtonEvent(ref triggerPressed, false, 0f));
                 EmitAlias(ButtonAlias.Trigger, false, 0f, ref triggerPressed);
             }
+            else
```

Comment:
do these need to be in the else? would this mean they don't trigger on the initial press or on the final release?